docs/setup/install.md

```diff
@@ -1,98 +1,250 @@
-# Installing Devtron
+[![Join Discord](https://img.shields.io/badge/Join%20us%20on-Discord-e01563.svg)](https://discord.gg/72JDKy4)
 
-## Install with Helm
+# Devtron Installation
 
-Devtron Helm chart is available at
+## Prerequisites
 
-[https://github.com/devtron-labs/devtron-installation-script/tree/main/charts](https://github.com/devtron-labs/devtron-installation-script/tree/main/charts)
+You will need to be ready with following prerequisites before Devtron installation
+ - A Kubernetes cluster (preferably K8s 1.16 or above) created on AWS (EKS or KOPS) or on AZURE (AKS or KOPS). Check [Creating a Production grade EKS cluster using EKSCTL](https://devtron.ai/blog/creating-production-grade-kubernetes-eks-cluster-eksctl/)
```

Comment:
can we simplify this.

Reply:
Simplify the words or Creation of cluster?

Reply:
our system is not dependent on how k8s has been created. the prerequisite for us is `k8s>= 16 &&. aws/azure`